feature-libs/product/configurators/common/components/update-message/configurator-update-message.component.html

```diff
@@ -1,9 +1,4 @@
-<div class="wrapper">
-  <div
-    id="cx-configurator-update-message"
-    [class.visible]="hasPendingChanges$ | async"
-  >
-    <cx-spinner></cx-spinner>
-    <strong>{{ 'configurator.header.updateMessage' | cxTranslate }}</strong>
-  </div>
+<div id="cx-update-message" [class.visible]="hasPendingChanges$ | async">
```

Comment:
Do we _really_ need to use id? Can we use simply css class?

Reply:
No we don't. I removed the _ID_
I just thought about our cypress tests that are waiting until the update message disappears